man/systemd.link.xml

```diff
@@ -1140,6 +1140,23 @@ Name=dmz0</programlisting>
       <filename>99-default.link</filename>, i.e. it needs a numerical prefix, to have any effect at all.</para>
     </example>
 
+    <example>
+      <title>(Re-)applying a .link file to an interface</title>
+
+      <para>If you create a new .link file or modify an existing .link file, to make the new settings applied
+      to the matching interface, you need to call something like the following:</para>
```

Comment:
"After a new .link file has been created, or an exisiting .link file modified, the new settings may be applied to the matching interface with the following commands:"